generators/server/templates/build.gradle.ejs

```diff
@@ -81,6 +81,19 @@ if (project.hasProperty("war")) {
     apply from: "gradle/war.gradle"
 }
 
+if (project.hasProperty("gae")) {
+    apply plugin: 'maven'
```

Comment:
The `maven` plugin is deprecated and `maven-publish` should be used. Is is required to add the plugin here? We have that in the global plugin definition. Does the `maven-publish` offer all functions here?

Reply:
@atomfrede: You are correct. The problem is that the [Spring Boot Thin Launcher](https://github.com/spring-projects-experimental/spring-boot-thin-launcher) doesn't support this new `maven-publish` plugin. It's mentioned in their [README.md](https://github.com/spring-projects-experimental/spring-boot-thin-launcher/blob/master/README.md). This is the reason I had to include this older plugin. :cry: